beacon-chain/rpc/eth/beacon/pool.go

```diff
@@ -307,6 +308,25 @@ func (bs *Server) SubmitVoluntaryExit(ctx context.Context, req *ethpbv1.SignedVo
 	return &emptypb.Empty{}, nil
 }
 
+// BroadcastBLSToExecutionChanges broadcasts all existing BLS changes in the
+// operations pool.
```

Comment:
you responded on slack but could you add some use-case comments here as the reason for having this endpoint is less obvious.